pyanaconda/core/kernel.py

```diff
@@ -134,12 +137,23 @@ def getbool(self, arg, default=False):
         :return: a boolean value of the argument
         """
         result = default
-        for a in self:
+        for a in self._data:
             if a == arg:
-                if self[arg] in ("0", "off", "no"):
+                if self._data[arg] in ("0", "off", "no"):
                     result = False
                 else:
                     result = True
             elif a == 'no' + arg:
                 result = False  # XXX: should noarg=off -> True?
         return result
+
+    def get(self, *args):
```

Comment:
The method `get` has only one optional argument `default=None`, so I think you can explicitly define it here.